src/main/kotlin/no/skatteetaten/aurora/gobo/integration/imageregistry/ImageRegistryServiceBlocking.kt

```diff
@@ -26,6 +31,28 @@ class ImageRegistryServiceBlocking(
 
     private val objectMapper = jacksonObjectMapper()
 
+    fun resolveTagToSha(key: ImageTag): String {
```

Comment:
Probably should have a test?

Reply:
ApplicationWithLatestDigestQueryResolverTest ?